examples/experimental/dagster-airlift/dagster_airlift/in_airflow/mark_as_migrating.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from typing import Any, Dict
+
+from airflow import DAG
+
+from ..migration_state import AirflowMigrationState
+
+
+def mark_as_dagster_migrating(
+    *,
+    global_vars: Dict[str, Any],
+    migration_state: AirflowMigrationState,
+    logger: logging.Logger = logging.getLogger("dagster_airlift"),
+) -> None:
+    """Alters all airflow dags in the current context to be marked as migrating to dagster.
+    Uses a migration dictionary to determine the status of the migration for each task within each dag.
+    Should only ever be the last line in a dag file.
+
+    Args:
+        global_vars (Dict[str, Any]): The global variables in the current context. In most cases, retrieved with `globals()` (no import required).
```

Comment:
Worth a pointer to https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#loading-dags to show how we are just doing what airflow does